core/src/test/java/org/elasticsearch/common/settings/loader/YamlSettingsLoaderTests.java

```diff
@@ -52,28 +53,17 @@ public void testSimpleYamlSettings() throws Exception {
 
     @Test(expected = SettingsException.class)
     public void testIndentation() {
+        String yaml = "/org/elasticsearch/common/settings/loader/indentation-settings.yml";
         settingsBuilder()
-                .loadFromClasspath("org/elasticsearch/common/settings/loader/indentation-settings.yml")
-                .build();
+            .loadFromStream(yaml, getClass().getResourceAsStream(yaml))
+            .build();
     }
 
     @Test(expected = SettingsException.class)
     public void testIndentationWithExplicitDocumentStart() {
+        String yaml = "/org/elasticsearch/common/settings/loader/indentation-with-explicit-document-start-settings.yml";
         settingsBuilder()
-                .loadFromClasspath("org/elasticsearch/common/settings/loader/indentation-with-explicit-document-start-settings.yml")
+                .loadFromStream(yaml, getClass().getResourceAsStream(yaml))
                 .build();
     }
-
-    
-    @Test
-    public void testYamlSettingsNoFile() throws Exception {
-        String invalidResourceName = "org/elasticsearch/common/settings/loader/no-test-settings.yml";
-        try {
-            Settings defaultSettings = settingsBuilder().loadFromClasspath(invalidResourceName).build();
-            fail("For a not exiting file an exception should be thrown.");
-        } catch (Exception e) {
-            assertTrue(e instanceof SettingsException);
-            assertThat(e.getMessage(), equalTo("Failed to load settings from [" + invalidResourceName + "]"));
-        }
-    }
```

Comment:
Yes, this was intentional. This was just checking loadFromClasspath's behavior for a non-existent resource, but loadFromClasspath is gone now.